Library/Homebrew/extend/os/mac/extend/ENV/shared.rb

```diff
@@ -23,4 +23,12 @@ def no_weak_imports_support?
 
     true
   end
+
+  sig { returns(T::Boolean) }
+  def no_fixup_chains_support?
+    return false if !MacOS::CLT.version.null? && MacOS::CLT.version < "13.0"
+    return false if !MacOS::Xcode.version.null? && MacOS::Xcode.version < "13.0"
```

Comment:
It's possible someone has CLT/Xcode 12 installed but the other is 13, with `xcode-select` pointing to the newer one.

We could check `/usr/bin/ld -v` for version < 711.

Reply:
Done.